README.md

```diff
@@ -1,88 +1,88 @@
 # Project
 
-This extension adds a `project` object to the `planning` object.
+This extension adds a `project` object to the `planning` object to describe the infrastructure or Public-Private Partnership (PPP) project to which the contracting process is related.
 
-In OCDS, project information is nested under the [`planning.budget`](https://standard.open-contracting.org/latest/en/schema/reference/#budget) object. However, in some cases, budget management systems and project management systems are separate, and it might be important to separately specify:
+The identifier for the infrastructure or PPP project to which a contracting process is related ought to be disclosed using the `planning.project.id` field.
 
-* The amount reserved in the budget for a specific contracting process
-* The project the contract relates to, and the total value of that project
-* Sector classifications
-* Additional classifications
-* Project locations, with options for gazetteer or point locations
-
-This is particularly important in cases of Public-Private Partnerships and large infrastructure projects, where users might want to track all the contracting processes related to the large-scale project, and to understand the individual contracts in the context of their contracting process and overall project values.
+The `planning.budget.projectID` field ought not be used to disclose the identifier for an infrastructure or PPP project. This field is used to disclose the identifier for a project in the national budget to which the contracting process is related. Since projects in the national budget might include many individual infrastructure projects, it is necessary to disclose these identifiers separately.
```

Comment:
```suggestion
The `planning.budget.projectID` field ought not to be used to disclose the identifier for an infrastructure or PPP project. This field is used to disclose the identifier for a project in the national budget to which the contracting process is related. Since projects in the national budget might include many individual infrastructure or PPP projects, it is necessary to disclose these identifiers separately.
```

I found this a bit difficult to parse but couldn't necessarily think of any better way! I think this slight change helps...

Reply:
I've reworded in https://github.com/open-contracting-extensions/ocds_project_extension/pull/29/commits/64f268819aa19f9f8201146393147f4a9b246526, based on https://github.com/open-contracting/standard/issues/887#issuecomment-512523250

Reply:
great, I think that's clearer now